docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -253,22 +433,34 @@ The first data link in the list will be configured as your one-click data link.
 
 {{< video-embed src="/media/docs/grafana/panels-visualizations/canvas-one-click-datalink-.mp4" >}}
 
-## Panel options
+### Selected connection options
 
-{{< docs/shared lookup="visualizations/panel-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+You can style the selected connection using the following options:
 
-## Standard options
+- **Color** - Set the connection color.
+- **Size** - Control the size of the connection by entering a number in the **Value** field.
+- **Radius** - Add curve to the connection by entering a value to represent the degree.
+- **Arrow Direction** - Control the appearance of the arrow head. Choose from:
 
-{{< docs/shared lookup="visualizations/standard-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+  - **Forward** - The arrow head points in the direction in which the connection was drawn.
+  - **Reverse** - The arrow head points in the opposite direction of which the connection was drawn.
+  - **Both** - Adds arrow heads to both ends of the connection.
+  - **None** - Removes the arrow head.
 
-## Thresholds
+- **Line style** - Choose from the following line styles: **Solid**, **Dashed**, and **Dotted**.
 
-{{< docs/shared lookup="visualizations/thresholds-options-2.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+### Standard options
 
-## Value mappings
+{{< docs/shared lookup="visualizations/standard-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+### Value mappings
 
 {{< docs/shared lookup="visualizations/value-mappings-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
```

Comment:
worth noting that in the preview nothing appears here

Reply:
Thanks for the heads up. It works in the local preview, but I've raised this with the docs platform team.